docs/source/compression/compression_config_list.rst

```diff
@@ -1,11 +1,190 @@
+for example, pruning targets should have same output channel number when pruning output channel.
+
+This key usually be used on modules with add operation, i.e., skip connection.
+
+granularity
+^^^^^^^^^^^
+
+Used to control the granularity of the generated masked.
+
+``default``, ``in_channel``, ``out_channel`` and list of integer are supported:
+
+- default: The pruner will auto determine using which kind of granularity, usually consistent with the paper.
+- in_channel: The pruner will do pruning on the weight parameters 1 dimension.
+- out_channel: The pruner will do pruning on the weight parameters 0 dimension.
+- list of integer: Block sparse will be applied. For example, ``[4, 4]`` will apply 4x4 block sparse on the last two dimensions of the weight parameters.
+
+Note that ``in_channel`` or ``out_channel`` is not supported for input/output targets.
+``torch.nn.Embedding`` is special, it's output dimension on weight is 1, so if want to pruning Embedding output channel, please set ``in_channel`` for its granularity.
+
+apply_method
+^^^^^^^^^^^^
+
+By default, ``mul``. ``mul`` and ``add`` is supported to apply mask on pruning target.
+
+``mul`` means the pruning target will be masked by multiply a mask metrix contains 0 and 1, 0 represents masked position, 1 represents unmasked position.
+
+``add`` means the pruning target will be masked by add a mask metrix contains -1000 and 0, -1000 represents masked position, 0 represents unmasked position.
+Note that -1000 can be configured in the future. ``add`` usually be used to mask activation module such as Softmax.
+
+Quantization Setting Keys
+-------------------------
+
+quant_dtype
+^^^^^^^^^^^
+
+By default, ``int8``. Support ``int`` and ``uint`` plus quant bits.
+
+quant_scheme
+^^^^^^^^^^^^
+
+``affine`` or ``symmetric``. If this key is not set, the quantization scheme will be choosen by quantizer,
+most quantizer will apply ``symmetric`` quantization.
+
+granularity
+^^^^^^^^^^^
+
+Used to control the granularity of the target quantization, by default the whole tensor will use same scale and zero point.
+
+``per_channel`` and list of integer are supported:
+
+- ``per_channel``: Each (ouput) channel will have their independent scales and zero points.
+- list of integer: The integer list is the block size. Each block will have their independent scales and zero points.
+
 Each sub-config in the config list is a dict, and the scope of each setting (key) is only internal to each sub-config.
 If multiple sub-configs are configured for the same layer, the later ones will overwrite the previous ones.
 
-Common Keys in Config
----------------------
+Distillation Setting Keys
+-------------------------
+
+lambda
+^^^^^^
+
+A float number. The scale factor of the distillation loss.
+
+link
+^^^^
+
+A teacher module name or a list of teacher module names. The student module link to.
+
+apply_method
+^^^^^^^^^^^^
+
+``mse`` or ``kl``.
```

Comment:
better to add more descriptions for "apply_method"

Reply:
sure, will add in distillation pr.